litgpt/pretrain.py

```diff
@@ -105,7 +105,11 @@ def setup(
             quit()
 
     hparams = capture_hparams()
-    data = TinyLlama() if data is None else data
+    data = data
+    if data is None or data == 'TinyLlama':
+        data = TinyLlama()
+    elif data == 'MicroLlama':
+        data = MicroLlama()
```

Comment:
data here can't actually be a string. It's either None or the datamodule object (see type annotation). I'm curious why did you see the need to add it?